test/fixtures/rules/std_rule_cases/L003.yml

```diff
@@ -51,6 +55,45 @@ test_fail_tab_indentation:
     rules:
       indent_unit: tab
 
+# test_case_in_fn:
```

Comment:
What's this for? If there's an issue that needs to be addressed before enabling this, could you mention that in a comment?

Reply:
This is the main one Im working on. Without this the test "should have been passing". but some 3.7 issues are causing them to fail.